app/components/c-panel.vue

```diff
@@ -20,9 +17,13 @@
 </template>
 
 <script>
+  import eHeading from '@/components/e-heading';
```

Comment:
- [ ] e-heading is globally imported. You don't need to import it on the component level.